crates/chaindata/service/src/tasks/model_listener.rs

```
@@ -64,42 +64,120 @@ impl ModelListenerTask {
 
     #[allow(clippy::match_wildcard_for_single_variants)]
     async fn process_model(&self, model_data: RawToriiData) {
-        let model = Model::parse(model_data).expect("Error while parsing model data");
-        let result = match model.model {
+        let parsed = match Model::parse(model_data.clone()) {
+            Ok(model) => model,
+            Err(err) => {
+                warn!(
+                    "Skipping model that failed to parse ({}): {:?}",
+                    err,
+                    model_data.name()
+                );
+                return;
+            }
+        };
```

Definetly not happenning.  
  
Skipping models that failed parsing means that we will miss them forever. This is one of the core designs of an indexer, fail fast. I prefer something that is fixed in time to a version that only has half of the elements due to a bug